tests/unit/type/string.tcl

```diff
@@ -607,6 +607,40 @@ if {[string match {*jemalloc*} [s mem_allocator]]} {
         r set foo bar pxat [expr [clock milliseconds] + 10000]
         assert_range [r ttl foo] 5 10
     }
+
+    test "SET EXAT / PXAT Expiration time is expired" {
```

Comment:
Is there test coverage for `unlink` client rewrite scenario already?

Reply:
i guess there is no coverage for unlink, but the code is easy so think this is enough.